encoding/serialization.go

```diff
@@ -55,15 +64,25 @@ func (c *G1Commitment) UnmarshalJSON(data []byte) error {
 	}
 	c.X = g1Point.X
 	c.Y = g1Point.Y
+
```

Comment:
`UnmarshalJSON` seems to be unused. Remove it?

Reply:
I think it is used by dataapi, https://github.com/Layr-Labs/eigenda/blob/master/disperser/dataapi/server.go#L50